src/main.cpp

```diff
@@ -190,14 +216,76 @@ int main(const int argc, const char *argv[])
     // Shift insets so that they do not overlap
     cart_info.shift_insets_to_target_position();
 
+    // Scale equal area map up
+    double scale_factor = 10e3;
+    for (auto &[inset_pos, inset_state] : *cart_info.ref_to_inset_states()) {
+      inset_state.scale_inset(scale_factor);
+    }
```

Comment:
Instead of doing this, we can modify the `InsetState::normalize_inset_area` function. Inside that method, there is a variable called `scale_factor`. If you wish to always scale up coordinates, we can just multiply that variable by 1000.